workspace-authorization/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
   compile group: 'org.apache.commons',            name: 'commons-lang3',                  version: apacheCommonsLangVersion
   compile group: 'org.springframework.security',  name: 'spring-security-jwt',            version: '1.0.8.RELEASE'
   compile group: 'org.springframework.retry',     name: 'spring-retry',                   version: '1.2.0.RELEASE'
+  compile group: 'io.swagger', name: 'swagger-jersey2-jaxrs', version: "1.5.21"
```

Comment:
we have property for swagger version in gradle.properties (swaggerVersion=1.5.21
)